package.json

```diff
@@ -42,10 +39,9 @@
     "react-dom": "^0.14.9 || >=15.3.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.11.4",
     "babel-core": "^6.13.2",
     "babel-eslint": "^8.0.1",
```

Comment:
how did we end up on v8.x here? should be v7.x probably... v8.x uses babel 7 under the hood